cli/pkg/gitutil/gitutil.go

```diff
@@ -21,6 +23,36 @@ func CloneRepo(url string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-
 	return repoName, nil
 }
+
+func ExtractRemotes(projectPath string) ([]string, error) {
```

Comment:
This returns *refs*, but it should return *remotes*. 

I.e. it should just do `repo.Remotes()` and return their Configs